tests/plugins/speculation-rules/speculation-rules-helper-test.php

```diff
@@ -54,6 +54,36 @@ static function () {
 		);
 	}
 
+	public function test_plsr_get_speculation_rules_href_exclude_paths_with_mode() {
```

Comment:
How about adding `@covers` to these test methods?

Reply:
Updated in https://github.com/WordPress/performance/pull/1025/commits/74b976fffa5846fbad41b93ef7de450645f7b13e